pkg/ccl/backupccl/backup_processor.go

```diff
@@ -289,7 +290,7 @@ func runBackupProcessor(
 						StartTime:                           span.start,
 						EnableTimeBoundIteratorOptimization: useTBI.Get(&clusterSettings.SV),
 						MVCCFilter:                          spec.MVCCFilter,
-						TargetFileSize:                      storageccl.ExportRequestTargetFileSize.Get(&clusterSettings.SV),
+						TargetFileSize:                      batcheval.ExportRequestTargetFileSize.Get(&clusterSettings.SV),
```

Comment:
I'm going to come for this in a follow-up that splits off the setting used by backup to pick its _target_ size from the setting used by kvserver to enforce its _max_ size, so this backup->batcheval edge is temporary